woocommerce/woocommerce-ios
# Make FulfillmentVC Footer View Reusable

Ref #2768. 

I made a mistake in #2936, it turns out I could not use that for the button in #2768. But since it's done and that button is now reusable anyway, I'm leaving the PR open. 😅 

## Changes 

This PR transfers this button in `FulfillViewController.xib` to a reusable class, [`ButtonTableFooterView`](https://github.com/woocommerce/woocommerce-ios/blob/issue/2768-reuse-button-footerview/WooCommerce/Classes/ViewRelated/ReusableViews/ButtonTableFooterView.swift). 

<img src="https://user-images.githubusercontent.com/198826/95406905-d2f28780-08d8-11eb-8921-87dcca081c3f.png" width="240">

I then added a `ButtonTableFooterView` to the refund confirmation page. 

<img src="https://user-images.githubusercontent.com/198826/95407714-0209f880-08db-11eb-9bcc-56be117efa97.png" width="240">

## Testing

1. Navigate to an unfulfilled order.
2. Tap “Begin Fulfillment”. 
3. Tap “Mark Order Complete”. Confirm that the order status was changed to “Completed”. 
4. Navigate to another order.
5. Tap “Issue Refund”. 
6. Tap “Next”. 
7. Confirm there is a “Refund” button. This page is currently read-only so there's nothing else to do here. 🤪 

## Submitter Checklist
